tests/jballerina-unit-test/src/test/resources/test-src/statements/variabledef/variable-definition-stmt.bal

```diff
@@ -45,3 +45,17 @@ function updateVarValue (int v1, boolean v3, string v4, float v5) returns [int,
 
     return [i, b, s, f];
 }
+
+string str = "";
+function wildCardLocalVariables() returns string {
```

Comment:
Shall we test some module-level variables also?

Reply:
Added in tests/jballerina-unit-test/src/test/resources/test-src/types/var/var-ignore-negative.bal. Will add more cases